packages/notifications/src/styled/global-alert/StyledGlobalAlert.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor, DEFAULT_THEME, retrieveComponentStyles } from '@zendeskgarden/react-theming';
+
+import { IGlobalAlertProps } from '../../types';
+import { getStartingDirection } from './utility';
+
+type StyledGlobalAlertProps = IGlobalAlertProps & ThemeProps<DefaultTheme>;
+
+const COMPONENT_ID = 'notifications.global-alert';
+
+function colorStyles(props: StyledGlobalAlertProps) {
+  let borderColor = null;
+  let backgroundColor = null;
+  let color = props.theme.palette.white;
+
+  switch (props.type) {
+    case 'success':
+      borderColor = getColor(props.theme.colors.successHue, 700, props.theme);
+      backgroundColor = getColor(props.theme.colors.successHue, 600, props.theme);
+      break;
+    case 'error':
+      borderColor = getColor(props.theme.colors.dangerHue, 700, props.theme);
+      backgroundColor = getColor(props.theme.colors.dangerHue, 600, props.theme);
+      break;
+    case 'warning':
+      borderColor = getColor(props.theme.colors.warningHue, 400, props.theme);
+      backgroundColor = getColor(props.theme.colors.warningHue, 300, props.theme);
+      color = getColor(props.theme.colors.warningHue, 800, props.theme) as string;
+      break;
+    case 'info':
+    default:
+      borderColor = getColor(props.theme.colors.primaryHue, 300, props.theme);
+      backgroundColor = getColor(props.theme.colors.primaryHue, 200, props.theme);
+      color = getColor(props.theme.colors.primaryHue, 800, props.theme) as string;
+      break;
+  }
+
+  return css`
+    border-color: ${borderColor};
+    background-color: ${backgroundColor};
+    color: ${color};
+  `;
+}
+
+function sizeStyles(props: StyledGlobalAlertProps) {
+  const height = props.theme.space.base * 13;
+  const padding = props.theme.space.base * 2;
+  const paddingStart = props.theme.space.base * 4;
+
+  return css`
+    border-bottom: 1px solid;
+    padding: ${padding}px;
+    width: 100%;
+    min-height: ${height}px;
+    ${getStartingDirection(props, 'padding', `${paddingStart}px`)};
+
+    & > svg {
+      flex-shrink: 0;
+      margin-top: ${props.theme.space.base * 2.5}px;
+    }
+
+    & > div {
+      margin-top: ${props.theme.space.base * 2}px;
+      margin-bottom: ${props.theme.space.base * 2}px;
+    }
```

Comment:
From what I'm seeing, all of the child selection here (`svg`, `div`, `button`) indicates the need to re-work the entire layout styling for the `GlobalAlert` and it's subcomponents. In summary, the 52px default height of the component needs to top/bottom pad in order to accommodate 20px line-height text. Without this, we're getting a 53px default `GlobalAlert` along with a bunch of pixel fussing, which is no bueno.